### PR TITLE
Create file utility

### PR DESCRIPTION
**Pull request by memeramita**
<h2>Changes</h2>

1. Created FileUtils class to handle file-related functions.
2. Refactor the code to use FileUtils.

### CODE REVIEW

File: napari_allencell_annotator/util/file_utils.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        ----------
+        file_path : Path
+            Name of the file to check.
+
+        Returns
+        -------
+        bool
+            True if the file is supported.
+        """
+        if file_path is None:
```
Comment (yrkim98) [inline napari_allencell_annotator/util/file_utils.py]: dont need this anymore

File: napari_allencell_annotator/util/file_utils.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            return False
+        extension: str = file_path.suffix
+        if extension in SUPPORTED_FILE_TYPES:
+            return True
+        else:
+            return False
```
Comment (yrkim98) [inline napari_allencell_annotator/util/file_utils.py]: `return extension in SUPPORTED_FILE_TYPES`

**Review by yrkim98 — APPROVED** [review]

File: napari_allencell_annotator/util/file_utils.py (hunk 1 above)
Comment (memeramita) [reply napari_allencell_annotator/util/file_utils.py]: Removed in https://github.com/aics-int/napari-allencell-annotator/pull/42/commits/f921f259f5f79d7a9c6be7c103476c7f64d457da

File: napari_allencell_annotator/util/file_utils.py (hunk 2 above)
Comment (memeramita) [reply napari_allencell_annotator/util/file_utils.py]: fixed in https://github.com/aics-int/napari-allencell-annotator/pull/42/commits/72a379bb2d47d6f51a17446a99efeae4844c0323

**Pull request merged**